src/Symfony/Component/ErrorHandler/ErrorRenderer/ErrorRenderer.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\ErrorHandler\ErrorRenderer;
+
+use Symfony\Component\ErrorHandler\Exception\ErrorRendererNotFoundException;
+use Symfony\Component\ErrorHandler\Exception\FlattenException;
+
+/**
+ * Renders an Exception that represents a Response content.
+ *
+ * @see ErrorRendererInterface
+ *
+ * @author Yonel Ceruto <yonelceruto@gmail.com>
+ */
+class ErrorRenderer
+{
+    private $renderers = [];
+
+    /**
+     * @param ErrorRendererInterface[] $renderers
+     */
+    public function __construct(array $renderers)
+    {
+        foreach ($renderers as $renderer) {
+            if (!$renderer instanceof ErrorRendererInterface) {
+                throw new \InvalidArgumentException(sprintf('Error renderer "%s" must implement "%s".', \get_class($renderer), ErrorRendererInterface::class));
+            }
+
+            $this->addRenderer($renderer, $renderer::getFormat());
+        }
+    }
+
+    public function addRenderer(ErrorRendererInterface $renderer, string $format): self
+    {
+        $this->renderers[$format] = $renderer;
+
+        return $this;
+    }
+
+    /**
+     * Renders an Exception and returns the Response content.
+     *
+     * @param \Exception|FlattenException $exception An \Exception or FlattenException instance
+     * @param string                      $format    The request format (html, json, xml, etc.)
+     *
+     * @return string The Response content as a string
+     *
+     * @throws ErrorRendererNotFoundException if no renderer is found
+     */
+    public function render($exception, string $format = 'html'): string
+    {
+        if (!isset($this->renderers[$format])) {
+            throw new ErrorRendererNotFoundException(sprintf('No error renderer found for format "%s".', $format));
+        }
+
+        if (!$exception instanceof FlattenException) {
+            $exception = FlattenException::create($exception);
```

Comment:
shouldn't this call `createFromThrowable`?

Reply:
see https://github.com/symfony/symfony/pull/32221